src/execution/riscv64/simulator-riscv64.cc

```diff
@@ -1714,7 +1714,7 @@ void Simulator::DecodeRVRType() {
 #endif /* V8_TARGET_ARCH_64_BIT */
       // TODO: Add RISCV M extension macro
     case RO_MUL: {
-      set_rd(sext_xlen(rs1() * rs2()));
```

Comment:
Stuff like this was in there for a universal version of the simulator (both 32-bits and 64-bits). I guess it is good to remove it now since we only are supporting 64-bit currently and have separated out 32-bit, but at some point later, we may want to re-consolidate again. Thoughts?

Reply:
Presumably we'd just have the register type be 32 bits for the rv32 target?  Then we don't need to mess around with explicit extensions all over the place, we just let C handle it.